app/controllers/movies_controller.rb

```diff
@@ -39,6 +39,47 @@ def destroy
 
   def show
     @movie = Movie.find(params[:id])
+    @actors_not_in_movie =  Actor.all.where.not(id: Movie.find(params[:id]).actors)
```

Comment:
Extra space before `Actor`

Reply:
Is this line working fine as you are passing actors to id in the query.
Also create a before_action `set_movie` and remove above line ie, `@movie = Moview.find..`